src/app/Library/CrudPanel/Traits/Read.php

```diff
@@ -349,11 +349,9 @@ private function abortIfInvalidPageLength($value)
      */
     public function enableExportButtons()
     {
-        if (! backpack_pro()) {
-            throw new BackpackProRequiredException('Export buttons');
-        }
-
```

Comment:
This is important. Export is a PRO feature, so it should trigger an exception, instead of working.